src/components/NoteDialogComponents/NoteDialogNoteWrapper.scss

```
@@ -4,16 +4,25 @@
 $note-dialog__scrollbar-height: calc(100vh - #{$header__height} - #{$padding--default} - #{$margin--default} - #{$margin--large});
 
 .note-dialog__scrollbar {
-  max-height: calc(#{$note-dialog__scrollbar-height} - 100px);
-  overflow: auto;
-  margin-left: $margin--default;
+  max-height: calc(#{$note-dialog__scrollbar-height} - 206px);
+  width: calc(100vw - (2 * $column__border-width));
+  overflow-y: auto;
+  overflow-x: hidden;
   scrollbar-color: $color-middle-gray $color-white;
+  margin-bottom: $column__border-width;
+  padding-top: $padding--large;
+  padding-left: 10px;
```

`$padding--small: 8px;` if you want to use it instead:
```suggestion
  padding-left: $padding--small;
```

---

10px was chosen because the scrollbar in the right has a width of 10px and I want the wrapper to be centered